.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  collect_sdks:
+    runs-on: ubuntu-latest
+    outputs:
+      sdks: ${{ steps.sdks.outputs.sdks }}
+    steps:
+      - uses: actions/checkout@v4
+      - id: sdks
+        run: echo "sdks=$(cd sdks && ls -d * | jq --raw-input --slurp --compact-output 'split("\n")[:-1]')" >> ${GITHUB_OUTPUT}
+  
+  build:
+    runs-on: ubuntu-latest
+    needs: collect_sdks
+    strategy:
+      matrix:
+        sdks: ${{ fromJson(needs.collect_sdks.outputs.sdks) }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+      
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build SDK Docker Images
+        uses: docker/build-push-action@v6
+        with:
+          context: sdks/${{ matrix.sdks }}
+          push: false
+
+  battle:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build Docker Compose Services
+        run: docker compose build
+
+      - name: Run Docker Compose
+        run: docker compose up server
+
+      - name: Collect Test Results
+        uses: actions/upload-artifact@v3
+        with:
+          name: game-results
+          path: ./server/results/
```

Comment:
In your sample fork, this is not grabbing any files. I don't know what there would be to collect, but if there's something you were expecting, it's not there.

Reply:
Removed. It was there as an example for a blog post.